movie-mlops-project/config/config.yaml

```
@@ -0,0 +1,27 @@
+  xgboost_params:
+    objective: "reg:squarederror" 
+    n_estimators: 1000 
+    learning_rate: 0.05 
+    max_depth: 6 
+    subsample: 0.7 
+    colsample_bytree: 0.7 
+    random_state: 42 
+    n_jobs: -1 
+
+mlflow:
+  tracking_uri: "http://localhost:5001" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)